src/UCommerce.Transactions.Payments.Adyen/AdyenPaymentMethodService.cs

```diff
@@ -50,17 +51,23 @@ public override Payment Extract(HttpRequest httpRequest)
             var jsonObj = (JObject?)JsonConvert.DeserializeObject(contentJson);
             var reference = jsonObj?["notificationItems"]?[0]?["NotificationRequestItem"]?[PaymentReferenceKey]
                 ?.Value<string>();
+            var eventCode = jsonObj?["notificationItems"]?[0]?["NotificationRequestItem"]?[EventCodeKey]

```

Comment:
After adding this it would be great to update the summary with an information about this, this will look odd later on because it is not SOLID anymore, but there is a reason for it to not be.